crt_portal/cts_forms/templates/forms/complaint_view/index/complaints_table.html

```diff
@@ -108,23 +116,24 @@
               </td>
             </tr>
             {# this row is hidden by default #}
-            <tr id="tr-additional-{{ datum.report.id }}" hidden>
-              <td colspan="2"></td>
-              <td colspan="4">
-                <div class="td-quickview">
+            <tr class="tr-quickview" id="tr-additional-{{ datum.report.id }}" hidden>
+              {# All screen readers don't yet read the headers attribute correctly, so we've added a redundant aria-label #}
+              {# https://a11ysupport.io/tech/html/headers_attribute #}
```

Comment:
> It's very easy to blow past the summary row that was just opened

This makes sense to me for the expansion toggle - I've updated it so going from the first or second column down will enter the details row

For the details box, I don't think we can, or maybe we don't even want to, jump to it from other columns based on alignment. If someone has some rows expanded, but wants to scan through ID, for example, they might prefer "ID: 5059, ID: 5060, ID: 5061" vs "ID: 5059, Summary, ID: 5060, Summary, ...". Beyond that UX, technically, I don't see a way to tell it to jump from a group of the cells above to the colspan cell below.

Going from submitted down will jump into Details, but I think that's unavoidable so long as this exists as a table row, and VoiceOver jumps right over it if I try to set it up as not a row due to the way it navigates between table rows.

Reply:
So I think you're right about the details box. One thing to note though is that VoiceOver reads different headings depending on the direction you're moving through the table. This happens in Chrome, not Firefox. 

What happens is if you navigate into Personal Description cell from a cell above it the heading of the cell is read correctly. However, if you try to navigate from a cell adjacent to it, the table heading for that column is read. Not sure how we can fix this, since it might be down to how the browsers handle the headers attribute.

Reply:
Yep - iiuc it's not something we can fix. It's what the `headers` attribute was designed for, but not all readers support it yet. I've added a bit of redundancy in the meantime so it will at least read the correct heading once.